Assets/Scripts/Actors/BuddyStats.cs

```diff
@@ -13,10 +19,21 @@ void Awake()
 		_particles = GetComponentInChildren<ParticleSystem>();
 	}
 
+	void Update()
+	{
+		if ( Input.GetKeyDown( KeyCode.K ) )
+		{
+			Kill();
+		}
+	}
+
 	public void GiveResource( ActorPhysics actorPhysics, ResourceData resourceData )
 	{
-		currentStats++;
-		Emote( heartMaterial );
+		if ( _isAlive )
```

Comment:
Put a comment or maybe an console error message saying that we should revisit this and put an assert in here.